packages/astro/src/runtime/client/dev-toolbar/ui-library/window.ts

```diff
@@ -24,9 +24,18 @@ export class DevToolbarWindow extends HTMLElement {
 					position: fixed;
 					z-index: 999999999;
 					bottom: 72px;
+					box-shadow: 0px 0px 0px 0px rgba(19, 21, 26, 0.30), 0px 1px 2px 0px rgba(19, 21, 26, 0.29), 0px 4px 4px 0px rgba(19, 21, 26, 0.26), 0px 10px 6px 0px rgba(19, 21, 26, 0.15), 0px 17px 7px 0px rgba(19, 21, 26, 0.04), 0px 26px 7px 0px rgba(19, 21, 26, 0.01);
+				}
+
+				:host([data-placement="bottom-left"]) {
+					left: 16px;
+				}
+				:host([data-placement="bottom-center"]) {
 					left: 50%;
 					transform: translateX(-50%);
-					box-shadow: 0px 0px 0px 0px rgba(19, 21, 26, 0.30), 0px 1px 2px 0px rgba(19, 21, 26, 0.29), 0px 4px 4px 0px rgba(19, 21, 26, 0.26), 0px 10px 6px 0px rgba(19, 21, 26, 0.15), 0px 17px 7px 0px rgba(19, 21, 26, 0.04), 0px 26px 7px 0px rgba(19, 21, 26, 0.01);
+				}
+				:host([data-placement="bottom-right"]) {
+					right: 16px;
 				}
```

Comment:
Two roads here:
- Either this should be an official prop, that user can change using the normal ways (like they can change the style and sizes of buttons, for instance)
- or, this code should be private to the windows we create ourselves, so this CSS would be in utils/window.ts instead

I'd favour the former, since it's the most intuitive for users and unlock a bit of freedom on their end.

Reply:
Thanks! The two roads make more sense than the current approach. I'll give the first one a try 💪

Reply:
I replaced the `data-placement` attribute with the [`placement` property](https://github.com/withastro/astro/pull/10591/commits/25ef09147507979dd91cfd6e716787d6c1e40cb8)